### PR TITLE
VACMS-19516 Add container and toggle for Email Signup React widget

### PR DESCRIPTION
**Pull request by randimays**
## Summary
Add the CMS feature toggle and empty `<div>` for the React widget for the email signup form.

**Note**: Feature toggle is currently **ON** in staging and production which means when this code is deployed to prod, we should continue to see the current/existing email form and not the new design.

Screenshot 2024-11-04 at 10 58 02 AM

Screenshot 2024-11-04 at 12 27 35 PM

## Related issue(s)
Ticket: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/19516

## Testing done (locally and in Tugboat)
1. Run `content-build` on the `19516-email-signup-widget` branch
  1a. comment everything but `...getNodePageQueries(entityCounts)` out in `individual-queries.js` to speed up the build
  1b. Point your CMS in the `.env` to `DRUPAL_ADDRESS=https://staging.cms.va.gov`
2. Run `vets-website` on `main`
3. Go to `localhost:3002` and validate that the original/current `Sign up to get the latest VA updates` form appears at the bottom of the homepage
4. Update your `.env` to `DRUPAL_ADDRESS=https://cms-g1rglcuobmlvgewnhny9uo7f6vigntwi.demo.cms.va.gov/` In this Tugboat, the feature toggle is OFF
5. Validate that you can only see the Veteran banner photos on desktop because the vets-website React widget has not yet been merged, and the (empty) container for it is replacing the original/current form. On mobile, you will see a narrow light blue box because the Veteran photos will not show, but this is not a realistic scenario that we need to account for.

### Feature toggle is ON
Screenshot 2024-11-01 at 3 26 20 PM

### Feature toggle is OFF
Screenshot 2024-11-04 at 11 43 24 AM

## What areas of the site does it impact?
Homepage email signup area only.

## Acceptance criteria
- [x] When the feature toggle is **ON**, the existing form will show
- [x] When the feature toggle is **OFF**, the new form will show

### CODE REVIEW

File: src/site/includes/email-update-signup.drupal.liquid [hunk 1]
```diff
@@ -1,59 +1,61 @@
...
-              type="submit"
-              class="vads-u-width--full
-                    medium-screen:vads-u-width--auto
-                    vads-u-display--block vads-u-margin-bottom--2">
-              Sign up
-            </button>
-          </form>
-        </div>
-      </div>
+<div class="homepage-email-update-wrapper vads-u-background-color--primary-alt-lightest vads-u-padding-x--2p5 vads-u-padding-top--2p5">
```
Comment (randimays) [inline src/site/includes/email-update-signup.drupal.liquid:1]: I combined two divs for the container here and added the toggle checker with the conditional for the empty React widget. That changed some indentation but I did not change the code for the original/current email form or the Veteran banner.

File: src/site/assets/sass/style.scss [hunk 2]
```diff
@@ -80,6 +80,13 @@
   }
 }
 
+.homepage-email-update-wrapper {
```
Comment (randimays) [inline src/site/assets/sass/style.scss:83]: This was inadvertently removed from vets-website a while back (cross-repo style dependencies biting us), so I re-added it here for the Veteran photo banner.

File: src/site/includes/email-update-signup.drupal.liquid (hunk 1 above)
Comment (eselkin) [reply src/site/includes/email-update-signup.drupal.liquid:1]: Makes sense

**Review by eselkin — APPROVED** [review]
Looks good to me.

**Review by chriskim2311 — APPROVED** [review]
Tested locally, LGTM!

**Pull request merged**